src/me/actions/creators/index.ts

```diff
@@ -5,6 +5,7 @@ import {RemoteDataState} from 'src/types'
 export const SET_ME = 'SET_ME'
 export const SET_QUARTZ_ME = 'SET_QUARTZ_ME'
 export const SET_QUARTZ_ME_STATUS = 'SET_QUARTZ_ME_STATUS'
+export const SET_ALERTS_STATUSES = 'SET_ALERTS_STATUSES'
```

Comment:
still confused as to where this is being used

Reply:
thought i removed it. removing!

Reply:
done!